src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        int numberOfFigures = 5;
```

Comment:
let's create a constant for 5 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names